ballista/core/src/execution_plans/shuffle_writer.rs

```diff
@@ -98,8 +78,6 @@ pub struct ShuffleWriterExec {
     shuffle_output_partitioning: Option<Partitioning>,
     /// Execution metrics
     metrics: ExecutionPlanMetricsSet,
-    /// Maximum number of rows to return
-    limit: Option<usize>,
```

Comment:
Is this limit introduced somewhere else?

Reply:
It's irrelevant now. This was a hack we had to apply a global limit across different partitions executing on the same executor. But now all of those partitions should execute as a single plan with the limit already applied globally across them

Reply:
Hm yeah :)